src/net/sourceforge/kolmafia/swingui/GenericFrame.java

```diff
@@ -475,81 +467,6 @@ public void addCompactPane() {
     this.framePanel.add(this.sidepane, BorderLayout.WEST);
   }
 
-  public void addScriptPane() {
```

Comment:
I use this all the time. I don't care if scriptList is used in the Relay Browser, but I depend on it being on the right side of the GUI.